lib/rules.js

```diff
@@ -5,7 +5,7 @@ var Rules = function(param, rules) {
   this.rules = rules;
 };
 
-Rules.prototype.Error = function(message, rule, value) {
+Rules.prototype.Error = function(message, rule, value, args) {
```

Comment:
you don't need `args` here anymore because it is no longer used.

remove this, and the PR is done! thanks so much.